### ISSUE TITLE
"ECONNREFUSED - Connection refused by server"

### ISSUE THREAD

**Issue by horia-m** [ppareit/swiftp#175]
I just started to use swiftp, v. 1.24, and after using it twice without any problems (I was very happy), now I get the econnrefused error when trying to connect with filezilla, over wifi.
there were no changes at the wifi setup, firewall of the computer, phone, etc. 
what could be the case?
thanks a lot for your answer!

**Comment by Xavron** [comment]
Did you solve this problem as of yet?

I just used FileZilla with it a lot today with zero issues here besides my debugging and fixing Android 11+ storage code issues anyway which is a different thing.

The only thing I can think of is that you need to reboot your device with the app on it maybe. Something changed though :)

**Comment by Xavron** [comment]
Also, not seeing a v 1.24 anywhere. The latest is 3.1

**Comment by ppareit** [comment]
Yes, probably older version. Might be because the android version of the device is older.
